integration/experiment/system_characterization/sys_charac_platfom_sweeps.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+source ${HOME}/.geopmrc
+source ${GEOPM_SOURCE}/integration/experiment/system_characterization/sys_charac_utils.sh
+
+BOARD_POWER_STEP=200 
+
+DRAM_MIN_POWER=$(geopmread MSR::DRAM_POWER_INFO:MIN_POWER board 0)
+GPU_MIN_POWER=$(geopmread LEVELZERO::GPU_POWER_LIMIT_DEFAULT board 0)
+PKG_MIN_POWER=$(geopmread MSR::PKG_POWER_INFO:MIN_POWER board 0)
+BOARD_MIN_POWER=$(( ${DRAM_MIN_POWER}+${GPU_MIN_POWER}+${PKG_MIN_POWER} ))
+
+DRAM_MAX_POWER=$(geopmread MSR::DRAM_POWER_INFO:MAX_POWER board 0)
+GPU_MAX_POWER=$(geopmread LEVELZERO::GPU_POWER_LIMIT_MAX_AVAIL board 0)
+PKG_MAX_POWER=$(geopmread MSR::PKG_POWER_INFO:MAX_POWER board 0)
+BOARD_MAX_POWER=$(( ${DRAM_MAX_POWER}+${GPU_MAX_POWER}+${PKG_MAX_POWER} ))
```

Comment:
We should talk about this and if it's starting at a large enough value.

Reply:
Awaiting input from platform power architects

Reply:
For the records, example values on an HPC compute node - 
BOARD_MIN_POWER=4068
BOARD_MAX_POWER=5294

Reply:
I'm a bit surprised the MIN value isn't lower than 4k...